src/Symfony/Component/Yaml/Inline.php

```diff
@@ -682,6 +682,10 @@ private static function parseTag(string $value, int &$i, int $flags): ?string
             throw new ParseException(sprintf('The built-in tag "!%s" is not implemented.', $tag), self::$parsedLineNumber + 1, $value, self::$parsedFilename);
         }
 
+        if ($tag && !isset($value[$i])) {
```

Comment:
`'' !== $tag`? (for consistency with the next lines?)

Reply:
makes sense, changed everywhere